content/javascript/concepts/promise/terms/any/any.md

```diff
@@ -65,3 +65,32 @@ Promise.any(promises)
   })
   .finally(() => console.log('Operations on Promise.any() have finished.'));
 ```
+
+## Codebyte Example
```

Comment:
This codebyte is essentially the same as example 2 in the existing entry. We do not include duplicates in our entries, and ideally each example demonstrates some new functionality about the method, the implementation, special cases, etc. Please revise.

Reply:
Thank you for your feedback! I will change the example to demonstrate how Promise.all() is resolved when the first resolved promise is found.